src/app/core/services/portal.service.ts

```diff
@@ -322,4 +323,20 @@ getMunicipalities(): Observable<any[]> {
     sDate = d.getUTCFullYear() + "-" + (d.getUTCMonth() + 1) + "-" + (d.getUTCDate() + 1) + "T00:08:00.000Z";
     return sDate;
   }
+
+getTransmissionProfiles(): Observable<any[]> {
+ return this._portalApi.getTransmissionProfiles();
+}
+
+getTransmissionProfile(id: number): Observable<ITransmissionProfile>{
+  return this._portalApi.getTransmissionProfile(id);
+}
+
+addTransmissionProfile(transmissionProfile: any): Observable<any> {
+  return this._portalApi.addTransmissionProfile(transmissionProfile);
```

Comment:
Is this for saving of transmission profile. Can we name it to "createTransmissionProfile" instead?

Reply:
Applied.